diff --git a/components/component_updater/BUILD.gn b/components/component_updater/BUILD.gn
index cdd4003..96a4ed4 100644
--- a/components/component_updater/BUILD.gn
+++ b/components/component_updater/BUILD.gn
@@ -36,6 +36,7 @@ static_library("component_updater") {
     "//components/crx_file",
     "//components/update_client",
     "//components/version_info",
+    "//services/network/public/mojom",
     "//third_party/boringssl:boringssl",
     "//ui/base",
     "//url",
